docs/docs/en/guide/parameter/context.md

```diff
@@ -78,7 +78,7 @@ The result of Node_mysql is as follows:
 
 ![context-log03](../../../../img/new_ui/dev/parameter/context_log03.png)
 
-Even though output is assigned a value of 1 in Node_A's script, the log still shows a value of 100. But according to the principle from [parameter priority](priority.md): `Local Parameter > Parameter Context > Global Parameter`, the output value in Node_B is 1. It proves that the output parameter is passed in the workflow with reference to the expected value, and the query operation is completed using this value in Node_mysql.
+Even though output is assigned a value of 1 in Node_A's script, the log still shows a value of 100. But according to the principle from [parameter priority](priority.md): `Startup Parameter > Local Parameter > Parameter Context > Global Parameter`, the output value in Node_B is 1. It proves that the output parameter is passed in the workflow with reference to the expected value, and the query operation is completed using this value in Node_mysql.
```

Comment:
Looks like we have docs for `Local Parameter`, `Parameter Context` and `Global Parameter`. I think it will be better if you could add some brief explanations on what is `startup parameter` here, since `startup parameter` may cause confusion to users who are not familiar with DS.

Reply:
> Looks like we have docs for `Local Parameter`, `Parameter Context` and `Global Parameter`. I think it will be better if you could add some brief explanations on what is `startup parameter` here, since `startup parameter` may cause confusion to users who are not familiar with DS.

Sure! Add a new page or explain in this page, which one you think is better?

Reply:
> > Looks like we have docs for `Local Parameter`, `Parameter Context` and `Global Parameter`. I think it will be better if you could add some brief explanations on what is `startup parameter` here, since `startup parameter` may cause confusion to users who are not familiar with DS.
> 
> Sure! Add a new page or explain in this page, which one you think is better?

A new tab (page) sounds better to me, as we could keep things consistent : ) 
![image](https://user-images.githubusercontent.com/34905992/209704576-61a572ff-9e97-4514-9687-83434e3e9942.png)

Reply:
Sure we should add a new page about the startup param